src/helm/benchmark/scenarios/oasst1_scenario.py

```diff
@@ -0,0 +1,106 @@
+from typing import List, Dict, Any
+from datasets import load_dataset, Dataset
+
+from .scenario import CORRECT_TAG, Reference, Scenario, Instance, Input, TRAIN_SPLIT, VALID_SPLIT, Output
+
+
+class Oasst1Scenario(Scenario):
+    """
+    This scenario is based on the OpenAssistant Conversations Dataset (OASST1) released by LAION.
+    The dataset includes 66,497 human-generated, human-annotated assistant-style conversation trees
+    in 35 different languages. Each conversation tree has an initial prompt message as the root node, and
+    every node can have multiple child messages. In total, there are 161,443 messages in the dataset.
+
+    https://arxiv.org/pdf/2304.07327.pdf
+
+    Note that we are only using the initial prompt messages and their direct responses in this scenario.
```

Comment:
This is an important caveat.  say explicitly that we're not including the subsequent turns of the chat.